src/components/Menu/AuthMenuItem.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { removeToken } from '../../utils';
```

Comment:
i wouldn't really advocate doing this kind of import, its hard to trace where this function is defined even after visiting the index file in utils folder, for ease i would prefer importing it directly from the auth file
`import { removeToken } from '../../utils/auth`